haaga-helia-university-of-applied-sciences-harvard.csl

```diff
@@ -167,6 +167,8 @@
       </else-if>
       <else>
         <text term="no date"/>
+        <!-- no date cites need to be unique -->
+        <text variable="year-suffix" prefix=" "/>
```

Comment:
If you add year suffix explicitly in one place, you also have to explicitly add it to the actual date.

Reply:
removed the attribute. the prefix attribute was incorrect and shouldn't be there in the first place. thanks for noticing it!